client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -129,6 +130,7 @@ export function* createSite(
 				font_base: selectedFonts.base,
 				font_headings: selectedFonts.headings,
 			} ),
+			...options,
```

Comment:
ah yeah, what if we just set the option here instead of everywhere else?

Reply:
I didn't want to evaluate a query arg (or config flag) inside of the `createSite` action (which I'd like to be a more general-purpose tool) :thinking:

Reply:
Gotcha. Conceptually, it feels like middleware, since we always want this config option to be passed through to the API. (I think.)

Reply:
At second glance... I don't wanna over-engineer it, and _this_ store (and actions) is already a Gutenboarding-customized layer on top of the more generic site store found in `@automattic/data-stores`. I'll move the logic here.